src/fty_common_asset_types.cc

```diff
@@ -104,6 +106,9 @@ subtype_to_subtypeid (const std::string &subtype)
     else if(st == "sts") {
         return asset_subtype::STS;
     }
+    else if(st == "power-meter") {
```

Comment:
is dash a safe character?

Reply:
hey, that's a good question! ;)
I thought about converting, to be both more coherent with the other such assets (patchpanel, ...) and be sure WRT an extra dash, possibly used in some tokenization (not scanned the code to assert though).
Going to amend the needed PRs, thx for pointing @EldoreiJK